frontend/src/metabase/collections/components/Header/Header.unit.spec.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import { render, screen } from "@testing-library/react";
+
+import Header from "./Header";
+
+const collection = {
+  name: "Name",
+};
+
+it("should display collection name", () => {
+  render(<Header collection={collection} />);
+
+  screen.getByText(collection.name);
+});
+
+describe("description tooltip", () => {
+  const ariaLabel = "info icon";
+
+  describe("should not be displayed", () => {
+    it("if description is not received", () => {
+      render(<Header collection={collection} />);
+
+      expect(screen.queryByLabelText(ariaLabel)).not.toBeInTheDocument();
+    });
+  });
+
+  describe("should be displayed", () => {
+    it("if description is received", () => {
+      const description = "description";
+
+      render(<Header collection={{ ...collection, description }} />);
+
+      screen.getByLabelText(ariaLabel);
+    });
+  });
+});
+
+describe("permissions link", () => {
+  const ariaLabel = "lock icon";
+
+  describe("should not be displayed", () => {
+    it("when isAdmin is not passed", () => {
+      render(<Header collection={collection} />);
+
+      expect(screen.queryByLabelText(ariaLabel)).not.toBeInTheDocument();
+    });
+
+    it("when collection includes personal_owner_id", () => {
```

Comment:
I believe test-case names can be improved here. E.g. here we test how the Header behaves for personal and not-personal collections (they're distinguished by the `personal_owner_id` field). But I'm not sure it's clear for anybody who hasn't worked on collections yet. I'd propose using more behavior-focused test names like "should not be displayed for personal collections". One extra benefit: if at some point we'll change how personal collections are defined, we won't need to update the test name, only the implementation

The same applies for `when isPersonalCollectionChild is truthy/falsey` and `when collection.can_write is truthy/falsey`

Reply:
I changed most of them. Thank you. Please let me know if not ok.